interfaces/builtin/modem_manager.go

```diff
@@ -47,6 +47,10 @@ var modemManagerPermanentSlotAppArmor = []byte(`
 # For ioctl TIOCSSERIAL ASYNC_CLOSING_WAIT_NONE
 capability sys_admin,
 
+# For {mbim,qmi}-proxy
+unix (create,bind,listen,send,receive) type=stream,
```

Comment:
This allows all abstract and anonymous of type stream and is too lenient. We should at a minimum have a label match on this rule, but likely more. Can you paste denials you are seeing?